### PR TITLE
Added support for Microsoft.UI.Xaml dependency property

### PR DESCRIPTION
**Pull request by jeffreyxrh**
https://dev.azure.com/ceapex/Engineering/_workitems/edit/443724

### CODE REVIEW

File: mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs
Comment (TianqiZhang) [inline mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs:168]: Just curious, why do we use `Name` here but `FullName` everywhere else?

Comment (jeffreyxrh) [reply mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs]: Indeed this should also be `FullName`.

Comment (TianqiZhang) [reply mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs]: LGTM, waiting for Joel for the final review and approval.

**Review by joelmartinez — APPROVED** [review]
Thanks for the 👀 @TianqiZhang ... this lgtm @jeffreyxrh. We'll merge it soon when we prepare the next release.

**Pull request merged**
